src/main/java/com/dev/movietheatre/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -22,10 +22,12 @@ public void addSession(MovieSession movieSession, User user) {
         Ticket ticket = new Ticket();
         ticket.setMovieSession(movieSession);
         ticket.setUser(user);
-        ticketDao.add(ticket);
-        ShoppingCart shoppingCartByUser = shoppingCartDao.getByUser(user);
-        shoppingCartByUser.getTickets().add(ticket);
-        shoppingCartDao.update(shoppingCartByUser);
+        if (shoppingCartDao.getByUser(user) != null) {
+            ticketDao.add(ticket);
+            ShoppingCart shoppingCartByUser = shoppingCartDao.getByUser(user);
```

Comment:
let's remove `if()` and just switch lines 26 and 27

Reply:
done